documentation/service-rules/ecf1/financial-statements.md

```diff
@@ -0,0 +1,208 @@
+
+---
+
+## How declarations are assigned to a statement
+
+When a lead provider submits a declaration via the `POST /api/v[1,2,3]/participant-declarations` endpoint, the declaration is assigned to the next _output fee_ financial statement—provided the participant is fundable and the declaration is not a duplicate.
+
+The _next output fee statement_ is determined as the first statement where:
+
+- `output_fee` is `true`
+- `deadline_date` is in the future
+- Ordered by `deadline_date ASC`
+
+Lead providers can check submitted declarations using the `GET /api/v[1,2,3]/participant-declarations` endpoint.
+
+Contract managers and product team members can check by:
+
+- Logging in as a user with the `finance` role
+- Visiting the **DfE Finance** dashboard
+- Navigating to the **Search participant data** page
+- Entering the participant ID or declaration ID
+- Scrolling down to the **Declarations** section, which lists all submitted declarations
+
+---
+
+## How statements are generated
+
+Financial statements are generated dynamically when visiting the **View financial statements** page in the **DfE Finance** dashboard.
+
+The following data is used to generate a statement:
+
+- Declarations (including clawbacks)
+- Call off contract
+- Mentor call off contract
+- Adjustments
+
+Statement values are calculated using the `Finance::ECF::StatementCalculator`.
+
+There are two versions of statements:
+
+- **Cohorts up to 2024** – values for ECTs and Mentors are combined
+- **Cohorts from 2025 onwards** – values for ECTs and Mentors are separated
+
+Declarations are submitted by lead providers. Call off contract and mentor call off contract information is provided by contract managers. Adjustments are also added by contract managers.
+
+---
+
+## How to view statements
+
+To view a financial statement:
+
+- Log in as a user with the `finance` role
+- Visit the **DfE Finance** dashboard
+- Go to the **View financial statements** page
+- Choose a lead provider
+- The financial statement will be displayed
+
+You can use the dropdowns to filter and view different statements:
+
+- Select a lead provider
+- Select a cohort
+- Select a statement month
+- Click the **View** button
+
+---
+
+## Downloading declarations (granular view)
+
+The statement page includes a **Download declarations (CSV)** link. This allows users to download a full list of all declarations used in the statement.
+
+The CSV includes the following fields:
+
+- Participant ID
+- Participant Name
+- TRN
+- Type
+- Mentor Profile ID
+- Schedule
+- Eligible for Funding
+- Eligible for Funding Reason
+- Sparsity Uplift
+- Pupil Premium Uplift
+- Sparsity and PP
+- Lead Provider Name
+- Delivery Partner Name
+- School URN
+- School Name
+- Training Status
+- Training Status Reason
+- Declaration ID
+- Declaration Status
+- Declaration Type
+- Declaration Date
+- Declaration Created At
+- Statement Name
+- Statement ID
+- Uplift Payable
+
+---
+
+## How to authorise statements for payment
```

Comment:
Is it worth mentioning how statements get marked as payable? I think its a job we run periodically?